examples/discord_bots/delete_warn_discord_bot.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# pylint: disable=duplicate-code
+# this is an example - duplication for emphasis is desirable
+# Aims to expose the full capabilities of this discord bot framework
+
+from __future__ import annotations
+
+from typing import Any, Dict, Set, Type
+
+import logging
+
+from mewbot.api.v1 import Trigger, Action
+from mewbot.core import InputEvent, OutputEvent, OutputQueue
+from mewbot.io.discord import (
+    DiscordInputEvent,
+    DiscordMessageCreationEvent,
+    DiscordMessageDeleteInputEvent,
+    DiscordOutputEvent,
+)
+
+
+class DiscordAllCommandTrigger(Trigger):
+    """
+    Nothing fancy - just fires whenever there is a DiscordTextInputEvent - of any type.
```

Comment:
Probably wants modifying if that's no longer the case? Or does the event trigger, and then it gets used on some instances?

Reply:
No - I think that's just a stale comment. Will update.

Reply:
Updated examples and comments to reflect new variable names - some more refactoring of the examples for clarity.